client/my-sites/site-settings/controller.js

```diff
@@ -91,6 +91,7 @@ export function importSite( context, next ) {
 		page.replace( context.pathname, {
 			engine: get( context, 'query.engine' ),
 			siteUrl: get( context, 'query.from-site' ),
+			signup: get( context, 'query.signup' ),
```

Comment:
`signup` sounds a bit vague. `isFromSignup` or `fromSignup` might show more meaning, what do you reckon?